### DISCUSSION TITLE
Gets to a blank screen on Windows build 22526

### DISCUSSION THREAD

**Discussion by Dasun9518** [valinet/ExplorerPatcher#637] [General]
Latest Explorer Patcher gets to a blank screen on Windows build 22526, But the previous update of it works fine.

**Comment by valinet** [comment]
Isn’t this: https://github.com/valinet/ExplorerPatcher/discussions/634 the same report? “Previous version” means previous Insider build of Windows, I presume, not EP, right?

**Reply by Dasun9518** [reply]
I just looked at that report and yes, It's the exact same problem.
The latest version of EP (22000.376.41) worked with the previous insider build of Windows without any issues. But it doesn't work with the latest windows (Build 22526.rs_prerelease) update. So I had to install an older version of EP (2200.348.39.1) which worked fine.

**Reply by valinet** [reply]
So it’s not the same report. Anyway, I understand, so presumably the previous stable version of EP works. Interesting, I wonder what is the problem, again…

You could try upgrading to the next pre-releases after that and see when it starts not working. With that determined, maybe we could identify the commit and exactly what in the commit changed and broke it.

**Reply by Dasun9518** [reply]
EP tried to update the program several times by itself causing that black screen issue. So, I had to turn off the automatic update and reinstall the EP 2200.348.39.1. That was the problem. But after turning off the updates, there's no other issue as of right now. 

Thank you for spending you time to address the issues and keeping this free. Wish you the best of luck!

**Reply by valinet** [reply]
The idea was to update manually to each subsequent pre-release and see where it breaks. Ofc if you disable the updates it won't update to the latest version. Yeah, I booted now a Windows Insider VM, I will update to this build and than do that...

**Comment by valinet** [comment]
Well, I cannot seem to reproduce this...

![image](https://user-images.githubusercontent.com/6503598/148506930-cf4f3057-1bc7-450c-9391-295d89e1460c.png)

**Reply by Dasun9518** [reply]
![Screenshot 2022-01-07 132124](https://user-images.githubusercontent.com/87459123/148510450-f81ebf2b-21b2-4f47-b78e-ac95ffd14c19.jpg)
I have the same Windows version too. But idk what's wrong in here. All the drivers are up to date as well.

**Reply by valinet** [reply]
Why don't you enable the console in Advanced section, then upgrade to latest version and see what it says there? or run under a debugger? Idk, I cannot guess, I am not a wizard, I tried reproducing, but as I said, on my setup works.

**Reply by Dasun9518** [reply]
https://user-images.githubusercontent.com/87459123/148512094-ef436b73-df1d-47bb-8d06-db66e07caa99.mp4

 This is basically what happens. And I can't access the console when that happens. But I'm really glad for your help so far. After all, this version of EP works fine for me. So I'll stick to this version for a while.

**Reply by valinet** [reply]
Okay. When you run the setup, wait 30 seconds, then kill it from Task manager if still there, then start Explorer manually (from Run new task, `explorer.exe`).

> So I'll stick to this version for a while.

I mean, since you posted this anyway, it would be best if this were debugged and fixed, if there's anything to fix.

**Reply by valinet** [reply]
Maybe it just gets hung restarting `explorer`, that used to happen more often in the past. I installed a clean new VM in VMware Workstation (which has a 3D accelerator) and tested the latest pre-release, that definitely works there.

**Reply by Dasun9518** [reply]
> Okay. When you run the setup, wait 30 seconds, then kill it from Task Manager if still there, then start Explorer manually (from Run new task, `explorer.exe`).

I tried doing that and the EP option window opened, but nothing happened to the blank screen. So I opened the console and this is what it was showing.

[Symbols] Reading symbols...
[Symbols] Hooking Win+C is not available for this build.
[Symbols] Failure in reading symbols for "C:\WINDOWS\system32\twinui.pcshell.dll

**Reply by valinet** [reply]
It's not the full output. Anyway, try with the latest pre-release, that should allow all symbols to be downloaded.
